server/src/domain/media/media.service.ts

```diff
@@ -330,7 +341,9 @@ export class MediaService {
     const allTargetsMatching = isTargetVideoCodec && isTargetAudioCodec && isTargetContainer;
     const scalingEnabled = ffmpegConfig.targetResolution !== 'original';
     const targetRes = Number.parseInt(ffmpegConfig.targetResolution);
+    const maxBitrate = Number.parseInt(ffmpegConfig.maxBitrate);
     const isLargerThanTargetRes = scalingEnabled && Math.min(videoStream.height, videoStream.width) > targetRes;
+    const isLargerThanTargetBitrate = bitrate >= maxBitrate;
```

Comment:
This should be `>` instead of `>=`. Also, this doesn't check the units. What unit does `results.format.bit_rate` use? You'd want to make sure the user input is converted to this. e.g. if it's in bps, but the user inputted in kbps ('k' at the end), you'd need to multiply the user input by 1000 to make the comparison.

Reply:
Okay, I will change it to >. 

I didn't know that the user can pass units. Thank you for pointing it out to me. I plan to fix everything later today.